services/agents/mysql_test.go

```diff
@@ -145,19 +149,22 @@ func TestMySQLdExporterConfigTablestatsGroupDisabled(t *testing.T) {
 			TLSKey:  "content-of-tls-key",
 		},
 	}
-	actual := mysqldExporterConfig(mysql, exporter, redactSecrets)
+	// TODO: Populate version information
+	agentVersion := &version.Parsed{}
+
+	actual := mysqldExporterConfig(mysql, exporter, redactSecrets, agentVersion)
 	expected := &agentpb.SetStateRequest_AgentProcess{
 		Type:               inventorypb.AgentType_MYSQLD_EXPORTER,
 		TemplateLeftDelim:  "{{",
 		TemplateRightDelim: "}}",
 		Args: []string{
 			"--collect.binlog_size",
 			"--collect.custom_query.hr",
-			"--collect.custom_query.hr.directory=" + pathsBase(pointer.GetString(exporter.Version), "{{", "}}") + "/collectors/custom-queries/mysql/high-resolution",
+			"--collect.custom_query.hr.directory=" + pathsBase(agentVersion, "{{", "}}") + "/collectors/custom-queries/mysql/high-resolution",
```

Comment:
let's set in expected paths that we expect to see instead of calling `pathsBase`

Reply:
Done